core/frontend/src/FrustumAnimator.ts

```diff
@@ -131,13 +131,18 @@ export class FrustumAnimator implements Animator {
 
   /** @internal */
   public animate() {
```

Comment:
Does GlobeAnimator need this in its animate() method as well?

Reply:
It could use it too, but I have not personally seen any demand for it or use-cases for it.  I'm sure they exist, and we can add it when there is the demand.

If you disagree, I don't see an issue adding it now either.

Reply:
As I look more in depth at the code, it seems setting a globe animator does not provide user access (currently) to ViewAnimationOptions.
I say skip adding anything for this PR, but could potentially be good later.
Thanks.